frontend/src/layout/navigation-3000/components/TopBar.tsx

```diff
@@ -101,7 +104,12 @@ export function TopBar(): JSX.Element | null {
                     )}
                     <Here breadcrumb={breadcrumbs[breadcrumbs.length - 1]} isOnboarding={isOnboarding} />
                 </div>
-                <div className="TopBar3000__actions" ref={setActionsContainer} />
+                {featureFlags['METALYTICS'] && (
```

Comment:
Looking for feedback here

Reply:
The should work but you should do `featureFlags[FEATURE_FLAGS.METALYTICS]` - so it's stored in the constant file.

Reply:
We can pair on this in the AM so I can show you how to test locally.

Reply:
Also should this always show? Or just on specific resources that support it?

Reply:
At the moment, we show it for all resources. We might want to add some logic down the line to show in places where we actually have appropriate meta analytics to display to the user. This can be done by adding checks on the read query to see if it returns insights that are not null or zero perhaps?..

Reply:
Verified that the feature flag works!